database/migrations/2019_09_13_094526_create_post_tag_table.php

```diff
@@ -1,35 +0,0 @@
-<?php
-
-use Illuminate\Support\Facades\Schema;
-use Illuminate\Database\Schema\Blueprint;
-use Illuminate\Database\Migrations\Migration;
-
-class CreatePostTagTable extends Migration
-{
-    /**
-     * Run the migrations.
-     *
-     * @return void
-     */
-    public function up()
-    {
-        Schema::create('post_tag', function (Blueprint $table) {
-            $table->bigIncrements('id');
```

Comment:
incrementing id not necessary for this table. 

Make post_id + tag_id a unique index